### PR TITLE
Activity dialog for subspaces

### PR DESCRIPTION
**Pull request by bobbykolev**
### Description

The two activity lists were merged into a dialog (subspace level) as per request: https://github.com/alkem-io/client-web/issues/5750

The current data fetch is not ideal but fixing it should be part of a separate task (comments left).

### CODE REVIEW

File: src/domain/journey/common/Activity/ActivityDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { useRouteResolver } from '../../../../main/routing/resolvers/RouteResolver';
+import DashboardAllContributionsBlock from '../dashboardRecentContributionsBlock/DashboardAllContributionsBlock';
+
+export interface SubspacesListDialogProps {
+  open?: boolean;
+  onClose?: () => void;
+  journeyId: string;
+}
+
+const SubspacesListDialog = ({ open = false, journeyId, onClose }: SubspacesListDialogProps) => {
```
Comment (ccanos) [inline src/domain/journey/common/Activity/ActivityDialog.tsx:17]: `ActivityDialog` ?

**Review by ccanos — DISMISSED** [review]
just a small comment, the rest looks fine to me

File: src/domain/journey/common/Activity/ActivityDialog.tsx (hunk 1 above)
Comment (bobbykolev) [reply src/domain/journey/common/Activity/ActivityDialog.tsx:17]: thanks for spotting it 🙏

**Comment by bobbykolev** [comment]
> Please have a look at the comments

Sure, thanks for looking at it 🙏

**Pull request merged**